### DISCUSSION TITLE
Integration of new relic java agent in Keycloak 17

### DISCUSSION THREAD

**Discussion by Snehalssg** [keycloak/keycloak#12315] [Q&A]
Can new keycloak support new relic java agent in same way as older keycloak versions?

**Comment by wimdeblauwe** [comment]
How did you do it in older versions? (Might be interesting for me as I still use the legacy Docker container: https://github.com/keycloak/keycloak/discussions/12375)

**Reply by mohamed0123** [reply]
@wimdeblauwe You can check the older way 
https://keycloak.discourse.group/t/added-apm-agent-in-keycloak/7242/4
